### PR TITLE
feat: web i18n optimization

### PR DESCRIPTION
**Pull request by yacosta738**
🚀 Pull Request Checklist

## 📄 Description

This pull request introduces several changes across multiple files, focusing on improving test setups, refining configurations, and enhancing localization. Key updates include the removal of outdated guidelines, restructuring test cases for better readability and maintainability, and implementing localization enhancements.

### Removed Documentation:
* [`.junie/guidelines.md`](diffhunk://#diff-15b715fb17970ebe09e6e65b0048e687dceafab1466dc820ccb181e3831a5297L1-L68): Entire file removed, including project overview, technology stack, coding conventions, testing guidelines, and best practices. This may imply deprecation or a shift to a different documentation strategy.

### Configuration Updates:
* [`client/apps/landing-page/package.json`](diffhunk://#diff-60cfbc4fafd6132fdf8b1a234a5ec63416193f551136e066014934f73cbfbcf6L35-R43): Updated scripts to use `run-p` for parallel execution and added new test commands (`test:ui`, `test:coverage`). Also updated the `vue-sonner` dependency version to `^2.0.2`. [[1]](diffhunk://#diff-60cfbc4fafd6132fdf8b1a234a5ec63416193f551136e066014934f73cbfbcf6L35-R43) [[2]](diffhunk://#diff-60cfbc4fafd6132fdf8b1a234a5ec63416193f551136e066014934f73cbfbcf6L67-R69)
* [`client/apps/web/package.json`](diffhunk://#diff-c80d20675926e5e0359fa1d0d36009d21071a7699987467edb3026b1b28d5f9bL15-R17): Added `test:ui` and `test:coverage` commands for enhanced testing capabilities.

### Testing Improvements:
* [`client/apps/web/src/account/login/login.spec.ts`](diffhunk://#diff-4649e7950df8813e283239ddd07b7a1abdc3598d18dafd2c88bca495221ae497L5-R19): Refactored tests to mock `vue-router` instead of using a real router, improving test isolation. Enhanced test cases with `vi.waitFor` for asynchronous operations and added validations for edge cases. [[1]](diffhunk://#diff-4649e7950df8813e283239ddd07b7a1abdc3598d18dafd2c88bca495221ae497L5-R19) [[2]](diffhunk://#diff-4649e7950df8813e283239ddd07b7a1abdc3598d18dafd2c88bca495221ae497L51-R112) [[3]](diffhunk://#diff-4649e7950df8813e283239ddd07b7a1abdc3598d18dafd2c88bca495221ae497L102-R218)
* [`client/apps/web/src/account/register/register.spec.ts`](diffhunk://#diff-0809909a0973827167a37d507fae4154cceb4df2747ccecf2cfbe104091c6c38R8-R13): Simplified form validation tests and added mocks for `vue-i18n` and `vue-sonner`. Improved test readability and ensured proper handling of asynchronous operations. [[1]](diffhunk://#diff-0809909a0973827167a37d507fae4154cceb4df2747ccecf2cfbe104091c6c38R8-R13) [[2]](diffhunk://#diff-0809909a0973827167a37d507fae4154cceb4df2747ccecf2cfbe104091c6c38L36-R123) [[3]](diffhunk://#diff-0809909a0973827167a37d507fae4154cceb4df2747ccecf2cfbe104091c6c38L95-R172)
* [`client/apps/web/src/components/ribbon/ribbon.component.spec.ts`](diffhunk://#diff-5ca8fece33f4bbe1c8b721defbaf6a5f839b940d5406f503d083786f5272cdc6R33-R34): Added a check for the `ribbonOnProfiles` property in the `AuthStore` to ensure proper test coverage.

### Localization Enhancements:
* [`client/apps/web/src/components/UserNav.vue`](diffhunk://#diff-92c01533ffc39d1ddff2ca558b9fb0d44b860e63903969e39ca9eea00388ccb7R10-R11): Integrated `vue-i18n` for translating the logout label and removed hardcoded text. Improved dependency injection for the `AccountService`. [[1]](diffhunk://#diff-92c01533ffc39d1ddff2ca558b9fb0d44b860e63903969e39ca9eea00388ccb7R10-R11) [[2]](diffhunk://#diff-92c01533ffc39d1ddff2ca558b9fb0d44b860e63903969e39ca9eea00388ccb7R25) [[3]](diffhunk://#diff-92c01533ffc39d1ddff2ca558b9fb0d44b860e63903969e39ca9eea00388ccb7L34-R37) [[4]](diffhunk://#diff-92c01533ffc39d1ddff2ca558b9fb0d44b860e63903969e39ca9eea00388ccb7L144-R147)

### Task Completion:
* [`.kiro/specs/vue-i18n-optimization/tasks.md`](diffhunk://#diff-14e7440fbcba0e44b8e4d593a54d27987de3f50829b163ff0a6c41061c56c467L3-R3): Marked the creation of a simplified i18n configuration system as complete.

---

### 📚 Documentation Reference

- [X] This PR follows the conventions defined in `/docs/conventions/`.

  ```
  (Please check all that apply and update the list below as needed.)
  ```
  - [X] [project-guidelines.md](../docs/conventions/project-guidelines.md)
  - [X] [rest-api.md](../docs/conventions/rest-api.md)
  - [X] [controller-pattern.md](../docs/conventions/controller-pattern.md)
  - [X] [swagger.md](../docs/conventions/swagger.md)
  - [X] [i18n.md](../docs/conventions/i18n.md)
  - [X] [offline-first.md](../docs/conventions/offline-first.md)
  - [X] [security.md](../docs/conventions/security.md)
  - [X] [uuid-strategy.md](../docs/conventions/uuid-strategy.md)
  - [X] [deprecation-policy.md](../docs/conventions/deprecation-policy.md)

### CODE REVIEW

File: client/packages/utilities/src/range/range.spec.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { describe, test, expect } from "vitest";
+import { describe, expect, test } from "vitest";
```
[the remark this answers is not in the document]
Comment (yacosta738) [reply client/packages/utilities/src/range/range.spec.ts:1]: we use biome not ESLint/Prettier
